Automattic/jetpack
# Update site accelerator recommendation image

Replace image used for the site accelerator recommendation because it is already being used somewhere else

#### Changes proposed in this Pull Request:

This PR replaces the illustration used on the site accelerator recommendation

#### Other information:


#### Jetpack product discussion

P2: p1HpG7-h2d-p2

#### Does this pull request change what data or activity we track or use?

No it does not

#### Testing instructions:

1. Check out this branch via `git switch update/site-accelerator-recommendation-image`
2. Start your local environment
3. Go to `/wp-admin/admin.php?page=jetpack#/recommendations/site-accelerator`
4. Confirm that the new image is being used for this recommendation
![image](https://user-images.githubusercontent.com/65001528/191618991-382e1684-4d34-48dc-a8db-4a8ebde38501.png)
